Andre-Dinh/index.html.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+  <title>Five Four Club</title>
+</head>
+<body>
+    <header>
+      <h1>FiveFour|Club</h1>
+      <nav>
+        <ul>
+        <a href="">HOW IT WORKS</a>
+        <a href="">WHO WE ARE</a>
+        <a href="">WHAT YOU RECEIVE</a>
+        <a href="">GIFT CARDS</a>
+        <a href="">LOG IN </a>
+        <a href="">GET STARTED</a>
+       </ul>
+      </nav>
+    </header>
+    <main>
+      <section>
+          <h2>THE EASIEST WAY TO NEVER HAVE TO SHOP AGAIN </h2>
+          <h3>Effortless style created just for you,
+            delivered right yo your doorstep!</h3>
+          <button> GET STATRED</button>
```

Comment:
I would use an <a> tag here. Buttons should mostly be reserved for forms or when you are capturing user input.